src/oemof/solph/components/_generic_chp.py

```diff
@@ -101,36 +101,49 @@ class GenericCHP(network.Transformer):
     >>> ccet = solph.components.GenericCHP(
     ...    label='combined_cycle_extraction_turbine',
     ...    fuel_input={bgas: solph.flows.Flow(
-    ...        H_L_FG_share_max=[0.183])},
+    ...        custom_attributes={"H_L_FG_share_max": [0.183]})},
     ...    electrical_output={bel: solph.flows.Flow(
-    ...        P_max_woDH=[155.946],
-    ...        P_min_woDH=[68.787],
-    ...        Eta_el_max_woDH=[0.525],
-    ...        Eta_el_min_woDH=[0.444])},
+    ...        custom_attributes={
+    ...            "P_max_woDH": [155.946],
+    ...            "P_min_woDH": [68.787],
+    ...            "Eta_el_max_woDH": [0.525],
+    ...            "Eta_el_min_woDH": [0.444],
+    ...        })},
     ...    heat_output={bth: solph.flows.Flow(
-    ...        Q_CW_min=[10.552])},
+    ...        custom_attributes={"Q_CW_min": [10.552]})},
     ...    Beta=[0.122], back_pressure=False)
     >>> type(ccet)
     <class 'oemof.solph.components._generic_chp.GenericCHP'>
     """
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-
-        self.fuel_input = kwargs.get("fuel_input")
-        self.electrical_output = kwargs.get("electrical_output")
-        self.heat_output = kwargs.get("heat_output")
-        self.Beta = solph_sequence(kwargs.get("Beta"))
-        self.back_pressure = kwargs.get("back_pressure")
+    def __init__(
+        self,
+        fuel_input,
+        electrical_output,
+        heat_output,
+        Beta,
```

Comment:
Weird that this is highlighted differently. It's not reserved, though, is it?

Reply:
This is because the parameter is capitalised. According to our style guide, it should be lower case. I will adapt this.